pkgs/by-name/co/cobalt/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "cobalt";
+  version = "0.19.8";
+
+  src = fetchFromGitHub {
+    owner = "cobalt-org";
+    repo = "cobalt.rs";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-neOJ3UqRisCcyarRIXfHyl9nAe2Wl9IXVDNwIYEQYys=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-j2xmEoMV7lVhqj4lKWA3QdEDEGUpRlZc4ikZoDQJlB8=";
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = [ "--version" ];
+  versionCheckProgram = "${placeholder "out"}/bin/cobalt";
```

Comment:
```suggestion
```
Shouldn't be necessary.

Reply:
Here it's not but removing it breaks overrides if someone renames pname. I believe the hook inherits the binary name from pname.

Reply:
Why would anyone override pname? Let them deal with the versionCheckProgram if they do so.

Reply:
In case they wanted two versions. Anyways, I have fixed it.